controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -342,6 +344,20 @@ function execute($request) {
 				$submission->setCopyrightHolder(null, null);
 				$submission->setLicenseURL(null);
 			}
+			
+			$submission->setDatePublished($this->getData('datePublished'));
+			
+			$submissionDao = Application::getSubmissionDAO();
+			if ($request->getUserVar('submissionRevision')) {
+				$revision = (int)$request->getUserVar('submissionRevision');
+			} else {
+				$revision = $submissionDao->getLatestRevisionId($submission->getId(), $context->getId());
```

Comment:
Also here, as for article update: would it maybe be better to have the information within the object itself? -- I cannot overlook all the situations for versioning... -- Maybe it is not better and not efficient... but just asking... :-)